docs/user/custom-domains.rst

```diff
@@ -2,125 +2,41 @@ Custom Domains
 ==============
 
 Custom domains allow you to serve your documentation from your own domain.
+For instance, this could be ``docs.example.com``.
 This is great for maintaining a consistent brand for your documentation and application.
 
-By default, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
+*By default*, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
 
 * ``<slug>.readthedocs.io`` for |org_brand|
 * ``<slug>.readthedocs-hosted.com`` for |com_brand|.
 
-For example if you import your project and it gets the :term:`slug` ``example-docs``, it will be served from ``https://example-docs.readthedocs.io``.
+How does it work?
+-----------------
 
-.. contents:: Contents
-    :local:
+To use a custom domain, you enter the domain in your Read the Docs project's administration and update your DNS provider with a new DNS entry.
 
-Adding a custom domain
-----------------------
+These two actions are all that are needed. Once the DNS record has propagated, Read the Docs automatically issues an SSL certificate through Cloudflare and starts serving your documentation.
```

Comment:
Would a screenshot help here? Or a small workflow diagram or something? I find this section pretty confusing to follow.

Reply:
A diagram could be nice.